pagopa/io-app
# fix(IT Wallet): [SIW-1658] Remove digital version badge from eID card

## Short description
This PR fixes a regression which causes the display of the "Digital version" badge on eID cards 

## List of changes proposed in this pull request
- Removed "Digital version" badge from eID card
- Added tests

## How to test
Navigate to the wallet, check that the eID does not have the "Digital version" badge on it, both in the wallet and in the details screen
